ddtrace/debugging/_probe/remoteconfig.py

```diff
@@ -83,47 +101,73 @@ def _wrapper(*args, **kwargs):
     return _wrapper
 
 
+def _create_probe_based_on_location(args, attribs, line_class, function_class):
+    # type: (Dict[str, Any], Dict[str, Any], Type, Type) -> Any
+    if attribs["where"].get("sourceFile", None):
```

Comment:
not sure what you mean. the configuration only have "snapshotProbe_" as type, regardless of the where value.

What refactoring did you have in mind?

Reply:
Ah this was just a consideration. I don't know if there is anything much that we can do with the current model, unless we have a `type` field, or we can rely on some prefix, like `snapshotProbe_` to infer the type instead of looking for fields

Reply:
I can't think of a way to remove this if (only move it to a different location